docs/Contributing/FAQ.md

```diff
@@ -91,6 +91,8 @@ If you also have Fleetd running on hosts, it will need access to these API endpo
 * `/api/fleet/orbit/config`
 * `/api/fleet/orbit/device_token`
 * `/api/fleet/orbit/ping`
+* `/api/fleet/orbit/scripts/request`
+* `/api/fleet/orbit/scripts/result`
```

Comment:
@noahtalerman  or @rachaelshaw  is that something we'd want to highlight in the changelog, that there are new endpoints that need to be accessible from orbit (from the hosts)? Not sure if that's needed because we recommend opening up everything under `/api/fleet/orbit/*` here: https://fleetdm.com/docs/get-started/faq#what-api-endpoints-should-i-expose-to-the-public-internet , but just in case.

> For full Fleet Desktop functionality, `/api/fleet/orbit/*` and `/api/fleet/device/ping` must also be exposed.

Reply:
Ah, I was just about to check the docs too.

It looks like we want to update the docs to recommend opening up `/api/fleet/orbit/*` for Fleet Desktop AND scripts?

![Screenshot 2023-08-22 at 9 56 43 AM](https://github.com/fleetdm/fleet/assets/47070608/41bd5e9a-9446-42e6-95cf-6d6aa40afdc1)

I think it makes sense to also call this out in the release notes. cc @spokanemac 

Martin, if I'm right about the above, do you mind making the doc change? Thanks!

Reply:
Whoops @mna forgot to tag you^

Reply:
Yeah I think the key here is that we want to make it clear to users what they need to expose to run scripts and what they need to expose to use Fleet Desktop (two different tools). Even if it's the same endpoints.